tests/integration/cattletest/core/test_schema.py

```diff
@@ -17,3 +17,10 @@ def test_account_default_fields(admin_client):
     assert network.nullable
     assert network.create
     assert network.update
+
+
+def token_account_default_fields(token_client):
+    schema = token_client.schema
+    assert schema.types['token'] is not None
```

Comment:
You don't need to change this but the idiomatic way to say this is `'token' in schema.types` if token was not in schema.types it would raise a KeyError and not return None.